contrib/pkginstaller/Makefile

```diff
@@ -41,6 +41,8 @@ packagedir: podman_version package_root Distribution welcome.html
 	echo -n $(ARCH) > $(PACKAGE_DIR)/ARCH
 	cp ../../LICENSE $(PACKAGE_DIR)/Resources/LICENSE.txt
 	cp hvf.entitlements $(PACKAGE_DIR)/
+	mkdir $(PACKAGE_ROOT)/podman/config
+	cp ../../pkg/machine/ocipull/policy.json $(PACKAGE_ROOT)/podman/config/policy.json
```

Comment:
These need to be moved into the package_root target in order to be correctly packaged.